### ISSUE TITLE
override service account in Pub sub trigger

### ISSUE THREAD

**Issue by ssvaidyanathan** [GoogleCloudPlatform/application-integration-management-toolkit#234]

**Comment by ssvaidyanathan** [comment]
@meghangupte - released as part of v0.76.0-beta.1

run the following

```sh
export INTEGRATIONCLI_VERSION=v0.76.0-beta.1
curl -L https://raw.githubusercontent.com/GoogleCloudPlatform/application-integration-management-toolkit/main/downloadLatest.sh | sh -
integrationcli --version
```

You should see `integrationcli version 0.76.0-beta.1 date: 2024-04-17T19:39:52Z [commit: 8ec8101]`

**Comment by meghangupte** [comment]
Hi @ssvaidyanathan I tried it but unfortunately this is still not working. If I use the same service account in the overrides files instead of directly using in the the integration file, I get an error `Reason: User not authorized to perform this action..`
However the user has permissions since it works if I include it in the integration file
